### ISSUE TITLE
Hyper-parameters used in the tSNE plot

### ISSUE THREAD

**Issue by meghbhalerao** [VisionLearningGroup/SSDA_MME#9]
Dear Authors,
Thank you for making the code open source! I had a small question while reading the paper. I was wondering if you could let me know the approximate hyper-parameters used while getting the tSNE embeddings as given in the paper, i.e. the perplexity, number of iterations, learning rate among others. I am plotting the tSNE [using this function](https://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE.html). Please do let me know if it would be possible. 
Thanks,
Megh
